plugins/dev9ghzdrk/Linux/dev9ghzdrk.ui

```diff
@@ -69,6 +69,7 @@
         </child>
         <child>
           <object class="GtkComboBoxText" id="IDC_BAYTYPE">
+		 <property name="sensitive">False</property>
```

Comment:
Tabs. The file uses normal spaces. Same goes for the other changes in this file.